src/plugins/highcharts/renderer/helpers/tooltip/index.ts

```diff
@@ -249,7 +249,9 @@ export const formatTooltip = (
         cellsRenderers,
         useCompareFrom: data.useCompareFrom,
         isSelectedLine: true,
-        allowComment: selectedLineIndex > tooltip.lastVisibleRowIndex,
+        allowComment:
+            typeof tooltip.lastVisibleRowIndex === 'number' &&
+            selectedLineIndex > tooltip.lastVisibleRowIndex,
```

Comment:
It was broken broke because of `tooltip.lastVisibleRowIndex` sometimes had `null` value, and 
```
selectedLineIndex > tooltip.lastVisibleRowIndex // true
```